### ISSUE TITLE
Update steps and browser_handler.py

### ISSUE THREAD

**Issue by tatiana-perry** [algolia/docsearch-scraper#371]
I had some trouble getting this running. 
Since selenium does not work on the new versions of Firefox. I had to download a [previous version](https://support.mozilla.org/en-US/kb/install-older-version-of-firefox#w_i-still-want-to-downgrade-ae-where-can-i-get-the-previous-version). Then I started getting this error:

`selenium.common.exceptions.WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/01/z68fg4h17y7fd8hjq331qmzw0000gp/T/tmp0UXJjA/webdriver-py-profilecopy If you specified a log_file in the FirefoxBinary constructor, check it for details.`

In the `scraper/source/config/browser_handler.py` I needed to specify a location for the Firefox profile and [create a new one](https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles). 

In summary to get it running:
Firefox: 41.0
Mac OS: 10.12.6 
Create new user profile
Add profile to browser_handler.py

 ```
profile = webdriver.FirefoxProfile('/Users/{user_name}/Library/Application 
  Support/Firefox/Profiles/{custom_profile_name}')

```

**Comment by s-pace** [comment]
:wave: @tatiana-perry 

The recommended version [is described here](https://github.com/algolia/docsearch-scraper/blob/65f14d371872189ed163e13ef721046ed99d943e/scraper/dev/docker/Dockerfile.base#L26)

If you install this precise one (v45.0.2), everything should work fine

**Comment by tatiana-perry** [comment]
Thanks @s-pace I did get it working and your fix works also.
